CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to the LaunchDarkly git-flag-parser will be documented in th
 - `updateSequenceId` is now an optional parameter. If not provided, data will always be updated. If provided, data will only be updated if the existing `updateSequenceId` is less than the new `updateSequenceId`.
 - Flags with keys shorter than 3 characters are no longer supported.
 - lines are truncated after 500 characters.
+- project layout modified to follow patterns from https://github.com/golang-standards/project-layout
+- use `launchdarkly` docker hub namespace
```

Comment:
I haven't been updating this changelog with my PRs. Let's make sure it's up to date when we release. Also, recommend sentence case for each line item